modules/bitgo/src/v2/keychains.ts

```diff
@@ -9,6 +9,9 @@ import { TssUtils } from './internal/tssUtils';
 import { BlsUtils } from './internal/blsUtils';
 
 const { validateParams } = common;
+
+export type KeyType = 'tss' | 'independent' | 'blsdkg';
```

Comment:
Note to reviewers: this comes from https://github.com/BitGo/bitgo-microservices/blob/develop/packages/api-docs/shared/KeyType.yaml